docs/content/docs/dev/table/sqlClient.md

```diff
@@ -60,106 +59,72 @@ or explicitly use `embedded` mode:
 ./bin/sql-client.sh embedded
 ```
 
+See [SQL Client startup options](#sql-client-startup-options) below for more details.
+
 ### Running SQL Queries
 
-Once the CLI has been started, you can use the `HELP` command to list all available SQL statements.
-For validating your setup and cluster connection, you can enter your first SQL query and press the `Enter` key to execute it:
+For validating your setup and cluster connection, you can enter the simple query below and press `Enter` to execute it.
 
 ```sql
-SELECT 'Hello World';
+SELECT 
+  name, 
+  COUNT(*) AS cnt 
+FROM 
+  (VALUES ('Bob'), ('Alice'), ('Greg'), ('Bob')) AS NameTable(name) 
+GROUP BY name;
 ```
 
-This query requires no table source and produces a single row result. The CLI will retrieve results
-from the cluster and visualize them. You can close the result view by pressing the `Q` key.
-
-The CLI supports **three modes** for maintaining and visualizing results.
-
-The **table mode** materializes results in memory and visualizes them in a regular, paginated table representation.
-It can be enabled by executing the following command in the CLI:
+The SQL client will retrieve the results from the cluster and visualize them (you can close the result view by pressing the `Q` key):
 
 ```text
-SET 'sql-client.execution.result-mode' = 'table';
+                           name                  cnt
+                          Alice                    1
+                           Greg                    1
+                            Bob                    2
```

Comment:
Could you show this display example using `tableau` mode which I think the community is tend to make it as default mode in the future. The `tableau` mode is also more readable than `table` mode.

Reply:
Yes, good idea.
I'll also set the execution mode to batch, to avoid showing the `+I` `-U` and `+U` to keep first example as simple as possible